packages/playwright/src/reporters/html.ts

```diff
@@ -24,36 +24,39 @@ import { open } from 'playwright-core/lib/utilsBundle';
 import { mime } from 'playwright-core/lib/utilsBundle';
 import { yazl } from 'playwright-core/lib/zipBundle';
 
-import { formatError, formatResultFailure, internalScreen } from './base';
+import { CommonReporterOptions, formatError, formatResultFailure, internalScreen } from './base';
 import { codeFrameColumns } from '../transform/babelBundle';
 import { resolveReporterOutputPath, stripAnsiEscapes } from '../util';
 
 import type { ReporterV2 } from './reporterV2';
-import type { HtmlReporterOptions, Metadata } from '../../types/test';
+import type { HtmlReporterOptions as HtmlReporterConfigOptions, Metadata, TestAnnotation } from '../../types/test';
 import type * as api from '../../types/testReporter';
 import type { HTMLReport, Stats, TestAttachment, TestCase, TestCaseSummary, TestFile, TestFileSummary, TestResult, TestStep } from '@html-reporter/types';
 import type { ZipFile } from 'playwright-core/lib/zipBundle';
 import type { TransformCallback } from 'stream';
-import type { ReporterOptions } from './types';
 
 type TestEntry = {
   testCase: TestCase;
   testCaseSummary: TestCaseSummary
 };
 
-type Options = HtmlReporterOptions & ReporterOptions;
-
-type HtmlReportOpenOption = Exclude<Options['open'], undefined>;
+type HtmlReportOpenOption = NonNullable<HtmlReporterConfigOptions['open']>;
 const htmlReportOptions: HtmlReportOpenOption[] = ['always', 'never', 'on-failure'];
 
 const isHtmlReportOption = (type: string): type is HtmlReportOpenOption => {
   return htmlReportOptions.includes(type as HtmlReportOpenOption);
 };
 
+type HtmlReporterOptions = HtmlReporterConfigOptions & {
```

Comment:
Shouldn't this be `HtmlReporterConfigOptions & CommonReporterOptions` instead?

Reply:
Good call, bad merge.